{"name": "test_invalid_auth", "status": "passed", "start": 1743880677041, "stop": 1743880677103, "uuid": "78ebb1d0-3c96-427f-80d3-28c0e1080896", "historyId": "10bb5b361ef41c939a9774d69a07ab28", "testCaseId": "10bb5b361ef41c939a9774d69a07ab28", "fullName": "Practice.Practice_4.2.4.2.3.test_cases.TestCases#test_invalid_auth", "labels": [{"name": "parentSuite", "value": "Practice.Practice_4.2.4.2.3"}, {"name": "suite", "value": "test_cases"}, {"name": "subSuite", "value": "TestCases"}, {"name": "host", "value": "192.168.1.4"}, {"name": "thread", "value": "18434-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Practice.Practice_4.2.4.2.3.test_cases"}]}